packages/contracts/contracts/protocol/incentives/IncentivesController.sol

```diff
@@ -122,6 +121,11 @@ contract IncentivesController is
 
     for (uint256 i = 0; i < assets.length; i++) {
       address asset = assets[i];
+
+      if (_incentivizedAssets[asset].decimals == 0) {
```

Comment:
Would it be better to check if numRewards is 0? In theory, ERC20 tokens can have 0 decimals